src/module.js

```diff
@@ -760,6 +764,15 @@ export default class DailyIframe extends EventEmitter {
     return this._activeSpeaker;
   }
 
+  setActiveSpeakerMode(enabled) {
+    this._sendIframeMsg({ action: DAILY_METHOD_SET_ACTIVE_SPEAKER_MODE, enabled });
+    return this;
+  }
+
+  activeSpeakerMode() {
+    return this._activeSpeakerMode;
+  }
+
```

Comment:
Totally happy with whatever naming feels right to you, since you've thought about this more than I have. We have `getNetworkStats()` and `getInputDevices()` methods. We also have `iframe()` and `participants()`. I probably didn't think carefully about a convention for naming these methods, but it kind of feels like we have an ad hoc division between "complicated" getters (that are probably async), which are prefixed with `get`, and simple getters that return a (probably cached) state. Maybe.

Reply:
As far as I can tell it looks like your naming has been totally consistent so far—there's a `get` prefix if the method is async, and no prefix if it's not. Doesn't seem ad hoc to me :)